src/towncrier/test/test_build.py

```diff
@@ -1530,3 +1530,60 @@ def test_orphans_in_non_showcontent_markdown(self, runner):
 
         self.assertEqual(0, result.exit_code, result.output)
         self.assertEqual(expected_output, result.output)
+
+    @with_project(
+        config="""
+        [tool.towncrier]
+        package = "foo"
+        build_ignore_filenames = ["template.jinja", "CAPYBARAS.md"]
+        """
+    )
+    def test_invalid_fragment_names(self, runner):
+        """
+        When build_ignore_filenames is set, files with those names are ignored.
+        """
+        opts = ["--draft", "--date", "01-01-2001", "--version", "1.0.0"]
+        # Valid filename:
+        with open("foo/newsfragments/123.feature", "w") as f:
+            f.write("Adds levitation")
```

Comment:
Maybe just document the purpose of the file inside it's content.
Tehn there is no need for a separate comment.
```suggestion
            f.write("Adds levitation in valid filename.")
```